README.md

```diff
@@ -144,6 +144,8 @@ Wrapbox.run(class_name, method_name, args,
   launch_type: "EC2", # Available only for ECS runner. The "launch_type" value in the configuration is used if it is nil.
   task_role_arn: nil, # Available only for ECS runner. The "task_role_arn" value in the configuration is used if it is nil.
   execution_role_arn: nil, # Available only for ECS runner. The "execution_role_arn" value in the configuration is used if it is nil.
+  tags: nil, # Available only for ECS runner. The "tags" value in the configuration is used if it is not given.
```

Comment:
[nits] For consistency, I think you should change the clause "if it is nil" in all the sentences to "if it is not given", or change the clause "if it is not given" in the new lines to "if it is not given".